### PR TITLE
Kill Consul

### LINKED ISSUE #181: Get off Consul

### PR DESCRIPTION
**Pull request by alanhdu**
Close #181

### CODE REVIEW

**Comment by alanhdu** [comment]
@danrschlosser @thebrianzeng Look good? How are we going to deploy Density w/o Consul?

**Comment by alanhdu** [comment]
@danrschlosser @thebrianzeng Thoughts? If there are no comments, I'll merge this tonight.

**Comment by danrschlosser** [comment]
This looks solid to me, sorry for letting it slip through the cracks

File: density/config/flask_config.py [hunk 1]
```diff
@@ -27,77 +25,29 @@
...
-        if not val:
-            raise Exception(("no value found in Consul for key "
-                             "density/{}").format(consul_key))
+try:  # use local settings
+    for env_key, value in config.iteritems():
+        if not value:
+            config[env_key] = os.environ[env_key]
 
-    # mail settings
-    config['MAIL_SERVER'] = 'smtp.gmail.com'
```
Comment (thebrianzeng) [inline density/config/flask_config.py]: why are we getting rid of all the mail stuff?

**Comment by alanhdu** [comment]
So, the mail password seems to only live on Consul. When we get closer to deploying a new version, I can rewrite it so that it loads this stuff from an environment variable.

**Pull request merged**